silo/tests/test_views.py

```diff
@@ -225,9 +227,23 @@ def test_export_csv(self):
 
 class SiloViewsTest(TestCase):
     def setUp(self):
+        factories.ReadType(read_type='CustomForm')
         self.org = factories.Organization()
         self.tola_user = factories.TolaUser(organization=self.org)
         self.factory = APIRequestFactory()
+        self.silo_id = None
+
+    def tearDown(self):
+        if self.silo_id:
+            # Have to remove the created lvs
+            lvss = LabelValueStore.objects.filter(silo_id=self.silo_id)
+            for lvs in lvss:
+                lvs.delete()
+
+    def bugfix_django_messages(self, request):
```

Comment:
Make it protected: `_`. And if it's a specific Django bug fix, would be nice in a docstring to have a link to the Django ticket tracking to know the reason and the status about that. This way if the bug is solved in some Django version, we can delete this fast.